tasks/sgx.py

```diff
@@ -15,7 +15,10 @@ def do_configure(repo_url, write_file, key_url):
 
 
 @task
-def configure():
+def configure(ctx):
+    """
+    Configure Intel and Microsoft APT repos
+    """
```

Comment:
This is convenient but looks like it's ubuntu 18.04-specific, might be worth putting a warning in the `README` about OS and version dependencies, then including a link to the official instructions for anyone who wants to diverge from that.